### PR TITLE
feat: add openzeppelin-contracts test

### PR DESCRIPTION
**Pull request by gpBlockchain**
Execute the test case of openzepping-contract on the alphanet at 4:50 every day

### CODE REVIEW

File: .github/workflows/alphanet_openzeppelin_test.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: gw_alphanet_v1_openzeppelin_test
+
+
+on:
+  schedule:
+  - cron:  '50 20/24 * * *' # every day
```
Comment (Flouse) [inline .github/workflows/alphanet_openzeppelin_test.yml]: [`50 20/24 * * *`](https://crontab.guru/#50_20/24_*_*_*) may not work with every cron.

File: .github/workflows/alphanet_openzeppelin_test.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+jobs:
+  contract-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        net: ['testnet_v1']
```
Comment (Flouse) [inline .github/workflows/alphanet_openzeppelin_test.yml]:
```suggestion
        net: ['alphanet_v1']
```

Comment (Flouse) [reply .github/workflows/alphanet_openzeppelin_test.yml]: This matrix is not used yet.

**Comment by ShookLyngs** [comment]
Yes, like @Flouse said, the matrix "net" is not used yet.
Do we need to use matrix in the workflow?

**Review by keroro520 — APPROVED** [review]

**Pull request closed without merging**